pkg/clusterconfig/testdata/minimal-cluster-config.golden

```diff
@@ -1,3 +1,8 @@
+addons:
```

Comment:
Is this minimal cluster thing part of the now-deleted "testcluster"? In that case we can probably remove it at some point in the future.

Reply:
Yes, it is! i have created a ticket for 'the purge 2'